.circleci/config.yml

```diff
@@ -106,8 +109,9 @@ commands:
           command: |
             pip3 install twine
             export TWINE_USERNAME=__token__
-            export TWINE_PASSWORD=$PYPI_API_TOKEN
-            python -m twine upload --non-interactive --repository pypi dist/*
+            export TWINE_PASSWORD=<< parameters.pypi-api-token >>
+            echo $TWINE_PASSWORD
```

Comment:
Is not `TWINE_PASSWORD` a secret?
Is it ok to print?

Reply:
Sorry, I forgot revert (by force commit) `test` commit. fix at https://github.com/Idein/nnoir/commit/9b57d01f49028b4f72e6fdd509ec548ac9fe8982

CircleCI hide environment values [like this](https://app.circleci.com/pipelines/github/Idein/nnoir/320/workflows/6f52ba42-a341-4146-8b79-b4ffbf9815ee/jobs/379).